### ISSUE TITLE
SyntaxError in oleform.py with python3.6

### ISSUE THREAD

**Issue by ant1** [decalage2/oletools#260]
Hi,

With python3.6 I have this error in oleform.py:

```
  File "/usr/local/lib/python3.6/site-packages/oletools/oleform.py", line 121
    if UUIDS == (199447043, 36753, 4558, 11376937813817407569L):
                                                             ^
SyntaxError: invalid syntax
```

**Comment by decalage2** [comment]
Good catch. Indeed, long integer literals with a trailing "L" were used in old python versions, but they are not necessary since Python 2.5, and even illegal for Python 3.x. So I'll simply remove the L.
References:
- https://docs.python.org/2/reference/lexical_analysis.html#integer-and-long-integer-literals
- https://www.python.org/dev/peps/pep-0237/

**Comment by decalage2** [comment]
fixed in commit e2d05a8e6b71d2b9382bc71cf24cbf84bdf8b049
